### PR TITLE
Add HoC cert generation exception handling

### PR DESCRIPTION
**Pull request by daynew**
# Description
*Bug*: When a student enters a veeeeeery long name when generating their certificate, ImageMagick throws an exception and doesn't return a certificate.
*Fix*: Capture the exception when ImageMagick tries to render the name as an image. Return a certificate with only the first 50 characters of a long name
* Also added honey badger logging for unexpected image magick exceptions.

## Links
- [honeybadger](https://app.honeybadger.io/projects/34365/faults/57678018)

## Testing story
* Added unit test for the long name experienced in production and verified the test fails if this fix is not implemented.
* Manually tested on `./bin/dashboard-server` and http://localhost.code.org:3000/certificates

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately

### CODE REVIEW

**Comment by daynew** [comment]
Created https://codedotorg.atlassian.net/browse/FND-923 for figuring out the best practice for logging warnings.

File: pegasus/test/test_certificate_image.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require_relative '../../lib/cdo/graphics/certificate_image'
 require_relative 'fixtures/mock_pegasus'
+require 'honeybadger/ruby'
```
Comment (bethanyaconnor) [inline pegasus/test/test_certificate_image.rb]: why do you need to include this?

**Review by bethanyaconnor — APPROVED** [review]
Thanks for adding this! And thanks for including a test :)

File: pegasus/test/test_certificate_image.rb (hunk 1 above)
Comment (daynew) [reply pegasus/test/test_certificate_image.rb]: oops!

**Pull request merged**